Docker/prod/build_images.sh

```
@@ -1,24 +1,26 @@
-
 #!/bin/bash
 
 # Change directory to root directory for correct Docker Context
+cd "$(dirname "$0")"
```

Shellcheck

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)